src/lib/gssapi/krb5/init_sec_context.c

```diff
@@ -875,6 +878,18 @@ mutual_auth(
         *time_rec = ts_delta(ctx->krb_times.endtime, now);
     }
 
+    /* refuse to set channel bound flag by default */
+    ctx->gss_flags = ctx->gss_flags & (~GSS_C_CHANNEL_BOUND_FLAG);
+
+    /*
+     * only set CB success flag if it was specified as a ret flag, and
+     * if channel bindings were specified by the caller.
```

Comment:
We can't ever set the channel-bound flag in the initiator with the current krb5 mech, because there is no channel-bindings confirmation in the mutual auth response.  This commit shouldn't need to make any changes to init_sec_context.